src/stateless/ChipWithMultiselect.vue

```diff
@@ -13,11 +13,13 @@
                 :label="searchLabel"
                 :auto-reorder="false"
                 :is-searchable="isSearchable"
+                :init-search-query="initSearchQuery"
                 :can-select-and-clear-all="canSelectAndClearAll"
                 :can-clear-all="canClearAll"
                 :options="options"
                 :size="size"
                 theme="light"
+                @search="searchQueryChange"
```

Comment:
Multiselect never emits 'search' event.

Reply:
Yeah, because you removed it https://github.com/celtra/stateless-components/pull/56/commits/e9686cb59a171201458fc1640ab63fe2b9533dec here ;)